src/main/java/org/apache/commons/text/StrTokenizer.java

```diff
@@ -124,10 +124,12 @@
     private StrMatcher trimmerMatcher = StrMatcher.noneMatcher();
 
     /** Whether to return empty tokens as null. */
-    private boolean emptyAsNull = false;
+    private boolean emptyAsNull;
     /** Whether to ignore empty tokens. */
     private boolean ignoreEmptyTokens = true;
 
+    private static final String[] EMPTY_STRING_ARRAY = new String[0];
```

Comment:
-1, reuse ArrayUtils.

Reply:
True. Change it for ArrayUtils.EMPTY_STRING_ARRAY